build_runner/lib/src/generate/input_set.dart

```diff
@@ -6,67 +6,66 @@ import 'package:build/build.dart';
 import 'package:collection/collection.dart';
 import 'package:glob/glob.dart';
 
-final List<Glob> _defaultInclude = new List.unmodifiable([new Glob('**')]);
+/// A set of file paths to match inputs to for a builder.
```

Comment:
We should update this - it doesn't really match what the class represents any more.... in fact the class should probably be renamed as well - `InputMatcher`?

Reply:
Done